DOCS-2428: Merge autogen: bill,mltrain

andf-viam opened the pull request:
Merge `billing` and `mltraining` generated `app` content.

npentrel commented on docs/appendix/apis/ml-training-client.md:
```diff
@@ -67,145 +67,6 @@ Once you have instantiated an `MLTrainingClient`, you can run the following [API
-
-{{% /tab %}}
-{{< /tabs >}}
+{{< readfile "static/include/app/apis/generated/mltraining.md" >}}
```
```suggestion
{{< readfile "/static/include/app/apis/generated/mltraining.md" >}}
```

npentrel reviewed (approved):
I think possibly hold off on mltraining until we've turned on flutter
- `SubmitTrainingJob` has lost a lot of info on the python parameters, as well as flutter code snippet
- `CancelTrainingJob` has lost some info on the python parameters

andf-viam commented:
I've added overrides for pySDK descriptions where ours are better, and removed the one Flutter method (it was not a real method, but the underlying proto, my fault there, I led Sierra astray). Plus automation picked up new Py code samples too.
